### ISSUE TITLE
Custom Extensibility Provider not called

### ISSUE THREAD

**Issue by petecuttriss** [SharePoint/PnP-Partner-Pack#158]
Hi,

I've created a custom extensibility provider by inheriting from IProvisioningExtensibilityProvider.   I've created a console project which applies the associated template and successfully calls the custom provider.

When I reference the custom provider with the PnP Partner Pack ScheduledJob and then create a site via the PnP Partner Pack using the same template, the site is successfully created however the custom provider isn't called.   When I view the web job log, there is no log noting "Extensibility Providers" (which comes from the OfficeDevPnP.Core engine").

Am I correct in referencing my custom provider from the ScheduledJob project and publishing the ScheduledJob with the associated referenced dll or am I missing something?  Should I be referencing my custom extensibility provider from the Infrastructure project?

Thanks for any help.
Regards Pete

**Comment by petecuttriss** [comment]
Hi, I've managed to do some further debugging and testing.   I've upgraded the OfficeDevPnP.Core reference to the latest release at the time of writing: 2.8.1610.1.   I then noticed that IProvisioningExtensibilityProvider was deprecated in favour of IProvisioningExtensibilityHandler.   I've updated my code to use the new interface and it is now working successfully and my extensibility provider is correctly called and executes successfully.
